### ISSUE TITLE
Bug: When I open and edit file in canvas, the console always display it.

### ISSUE THREAD

**Issue by YIRU69** [mdelobelle/metadatamenu#211]
```
TypeError: Cannot read properties of null (reading 'basename')
    at enter (plugin:metadata-menu:11309:96)
    at e.iterate (app.js:1:501709)
    at e.Uc.iterate (app.js:1:1046086)
    at import_view.ViewPlugin.fromClass.decorations.buildDecorations (plugin:metadata-menu:11281:53)
    at new import_view.ViewPlugin.fromClass.decorations (plugin:metadata-menu:11260:31)
    at e.create (app.js:1:350083)
    at e.update (app.js:1:350513)
    at e.setState (app.js:1:440093)
    at t.set (app.js:1:1324208)
    at t.showEditor (app.js:1:1348776)
```

**Comment by mdelobelle** [comment]
Is it fixed with 0.3.10?

**Comment by YIRU69** [comment]
It fixed! Thanks sir
